config.json

```diff
@@ -358,6 +357,16 @@
         "mathematics",
         "iterators"
       ]
+    },
+    {
+      "uuid": "2ed6e171-04e1-b080-f383-a801b2564b3e0f4f1ee",
+      "slug": "grains",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
I think this could be unlocked by difference-of-squares because they're somewhat similar.